spec

```diff
@@ -583,21 +583,30 @@ endseg
 beginseg
     name "ovl_opening"
     include "build/src/overlays/gamestates/ovl_opening/z_opening.o"
-    include "build/src/overlays/gamestates/ovl_opening/z_opening_reloc.o"
+    include "build/src/overlays/gamestates/ovl_opening/ovl_opening_reloc.o"
 endseg
 
 beginseg
     name "ovl_file_choose"
+    include "build/src/overlays/gamestates/ovl_file_choose/z_file_nameset_PAL.o"
+    include "build/data/overlays/gamestates/z_file_nameset_PAL.data.o"
+    include "build/data/overlays/gamestates/z_file_nameset_PAL.rodata.o"
+    include "build/data/overlays/gamestates/z_file_nameset_PAL.bss.o"
     include "build/src/overlays/gamestates/ovl_file_choose/z_file_choose.o"
-    include "build/data/overlays/gamestates/z_file_choose.data.o"
     include "build/data/overlays/gamestates/z_file_choose.rodata.o"
-    include "build/data/overlays/gamestates/z_file_choose.bss.o"
     include "build/data/overlays/gamestates/z_file_choose.reloc.o"
 endseg
 
 beginseg
     name "ovl_kaleido_scope"
-    include "build/src/overlays/actors/ovl_kaleido_scope/z_kaleido_scope.o"
+    
+    include "build/src/overlays/actors/ovl_kaleido_scope/z_kaleido_collect.o"
+    include "build/src/overlays/actors/ovl_kaleido_scope/z_kaleido_debug.o"
+    include "build/src/overlays/actors/ovl_kaleido_scope/z_kaleido_equipment.o"
+    include "build/src/overlays/actors/ovl_kaleido_scope/z_kaleido_item.o"
+    include "build/src/overlays/actors/ovl_kaleido_scope/z_kaleido_map_PAL.o"
+    include "build/src/overlays/actors/ovl_kaleido_scope/z_kaleido_scope_PAL.o"
+    include "build/src/overlays/actors/ovl_kaleido_scope/z_lmap_mark.o"
     include "build/data/overlays/actors/z_kaleido_scope.data.o"
     include "build/data/overlays/actors/z_kaleido_scope.rodata.o"
     include "build/data/overlays/actors/z_kaleido_scope.bss.o"
```

Comment:
I would disagree with splitting the code files without splitting the data files accordingly.
Also from what I've seen in rodata, there should be at least 2 more file splits in kaleido_scope (before and after z_lmap_mark.c).